package.json

```diff
@@ -56,13 +56,13 @@
   ],
   "dependencies": {
     "d3": "^4.7.4",
-    "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "^1.0.3",
+    "ember-cli-babel": "^5.1.7",
+    "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-sass": "^5.6.0",
     "ember-computed-decorators": "0.3.0",
     "ember-concurrency": "0.7.19",
     "ember-hook": "^1.4.1",
-    "ember-prop-types": "^3.10.2",
+    "ember-prop-types": "^3.11.0",
```

Comment:
Decide whether this is a dev dependency or a dependency, because it's listed in both sections.